python/cm/issue.py

```diff
@@ -247,26 +247,59 @@ def check_hc(cluster):
             return False
     try:
         check_component_requires(shc_list)
+        check_binded_components(shc_list)
     except AdcmEx:
         return False
     return True
 
 
 def check_component_requires(shc_list):
+    def get_components_with_requires():
+        return [i for i in shc_list if i[2].component.requires]
```

Comment:
I suggest changing the name "i" to "shc", because "i" is a structure, not a number